src/common/sentry.ts

```diff
@@ -0,0 +1,24 @@
+/** Copyright © 2024-present P0 Security
+
+This file is part of @p0security/cli
+
+@p0security/cli is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3 of the License.
+
+@p0security/cli is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
+**/
+import * as Sentry from "@sentry/node";
+
+const ENABLED_ENVIRONMENTS = ["stage", "staging", "prod", "production"];
```

Comment:
I do have a question about how to best handle the environment in the CLI.

Reply:
We could still use the NODE_ENV environment variable but default to production.
We already have a bunch of configuration in `drivers/env.ts`. We could add 
```
environment: env.NODE_ENV ?? "production",
```
to that (does it have to be "prod" or "production"?) and configure our repos in with NODE_ENV=development in a `.env` file.